Xamarin.Forms.Platform.WPF/Renderers/ImageRenderer.cs

```diff
@@ -159,4 +159,72 @@ public Task<System.Windows.Media.ImageSource> LoadImageAsync(ImageSource imageso
 			return Task.FromResult<System.Windows.Media.ImageSource>(bitmapimage);
 		}
 	}
+
+	public sealed class FontImageSourceHandler : IImageSourceHandler
+	{
+		public Task<System.Windows.Media.ImageSource> LoadImageAsync(ImageSource imagesource, CancellationToken cancelationToken = default)
+		{
+			var fontsource = imagesource as FontImageSource;
+			var image = CreateGlyph(
+					fontsource.Glyph,
+					new FontFamily(new Uri("pack://application:,,,"), fontsource.FontFamily),
+					FontStyles.Normal,
+					FontWeights.Normal,
+					FontStretches.Normal,
+					fontsource.Size,
+					CreateBrush(fontsource.Color));
+			return Task.FromResult(image);
+		}
+
+		private static Brush CreateBrush(Color color)
+		{
+			var wpColor = System.Windows.Media.Color.FromArgb(
```

Comment:
You can use the extension method from here:
https://github.com/xamarin/Xamarin.Forms/blob/69171055d6b218d24747cfe5b7beb200344f5317/Xamarin.Forms.Platform.WPF/Extensions/ColorExtensions.cs#L17-L20

Reply:
Thanks @campersau . I have updated  PR with your suggestion